index.html

```diff
@@ -13,47 +13,59 @@
     <link href="css/bootstrap.min.css" rel="stylesheet">
     <link href="css/style.css" rel="stylesheet">
 
-  </head>
-  <body>
-
-    <div class="container-fluid">
-	<div class="row">
-		<div class="col-md-12">
-			<h3>
-				Sign In
-			</h3>
-		</div>
-	</div>
-	<div class="row">
-		<div class="col-md-12">
-			<form role="form" id="login-form">
-				<div class="form-group">
-					 
-					<label for="exampleInputEmail1">
-						Email address
-					</label>
-					<input type="email" class="form-control" id="exampleInputEmail1" name="email">
-				</div>
-				<div class="form-group">
-					 
-					<label for="exampleInputPassword1">
-						Password
-					</label>
-					<input type="password" class="form-control" id="exampleInputPassword1" name="pass">
-				</div>
-				<button type="submit" class="btn btn-primary">
-					Submit
-				</button>
-				<button type="submit" class="btn btn-primary">
-				Return
-			</button>
-			</form>
-		</div>
-	</div>
+</head>
+<body>
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md-12">
+            <h3>
+                Sign In
+            </h3>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <form role="form" id="login-form">
+                <div class="form-group">
+
+                    <label for="exampleInputEmail1">
+                        Email address
+                    </label>
+                    <input type="email" class="form-control" id="exampleInputEmail1" name="email">
+                </div>
+                <div class="form-group">
+
+                    <label for="exampleInputPassword1">
+                        Password
+                    </label>
+                    <input type="password" class="form-control" id="exampleInputPassword1" name="pass">
+                </div>
+                <div class="checkbox">
+
+                    <label>
+                        <input type="checkbox" check="checked" name="chkRem" id="checkbox" checked="checked"
+                               value="true"> Remember me
+                    </label>
+                </div>
+                <button type="submit" class="btn btn-primary">
+                    Log in
+                </button>
+            </form>
+        </div>
+    </div>
 </div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<!--===============================================================================================-->
```

Comment:
I forgot to pull request at the last time. so I put them together in this PR. I will separate them.  I put these comment lines since I was trying to make these JS file looked more readable. I can remove them